src/test/java/htsjdk/tribble/index/tabix/TbiEqualityChecker.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2019 The Broad Institute
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package htsjdk.tribble.index.tabix;
+
+import htsjdk.samtools.Bin;
+import htsjdk.samtools.BinningIndexContent;
+import htsjdk.samtools.Chunk;
+import htsjdk.samtools.LinearIndex;
+import htsjdk.samtools.util.BlockCompressedFilePointerUtil;
+import htsjdk.samtools.util.BlockCompressedInputStream;
+import htsjdk.tribble.index.IndexFactory;
+import org.testng.Assert;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+public class TbiEqualityChecker {
+
+    private Path vcfFile;
+    private Path tbiFile1;
+    private Path tbiFile2;
+
+    private BlockCompressedInputStream blockStream;
+
+    public static void assertEquals(Path vcfFile, Path tbiFile1, Path tbiFile2, boolean identical)
```

Comment:
could you add documentation explaining the case the identical parameter is for

Reply:
Done